pkg/manager/impl/validation/validation.go

```diff
@@ -59,6 +59,30 @@ func ValidateIdentifier(id *core.Identifier, expectedType common.Entity) error {
 	return nil
 }
 
+// Validates that all required fields for an identifier are present.
+func ValidateNamedEntityIdentifier(id *admin.NamedEntityIdentifier) error {
```

Comment:
Please add tests for all these new methods!

Reply:
Done!